server/WHO.postman_collection.json

```diff
@@ -0,0 +1,174 @@
+{
```

Comment:
Possible to replace tabs with double spaces? Then it would show better on GitHub during review and editing.

Reply:
Done.

Reply:
Done.

Reply:
@rjhuijsman - when an issue is "Done", just click on the "Resolve conversation" in GitHub